src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.js

```diff
@@ -232,6 +236,11 @@ function ComposerWithSuggestions({
             setValue(newCommentConverted);
             if (commentValue !== newComment) {
                 const position = Math.max(selection.end + (newComment.length - commentRef.current.length), cursorPosition || 0);
+
+                if (isIOSNative) {
```

Comment:
Sounds reasonable to me as well, however I am afraid that since this is handled right in the middle of a callback, then applying the Base+props approach would introduce some code duplication, since the surroundings would need to be copied. Alternatively, passing props like `preUpdateComment` and `postUpdateComment` in `onChangeText` would do the job, but I think this could overcomplicate the codebase - I'll leave the decision up to the C+ in that case. Thanks for the idea!

Reply:
Resolved in 062265efd0de45f311da38c81f0a91333ac8ec80